osu.Game/Screens/Play/HUD/ArgonSongProgressBar.cs

```diff
@@ -33,6 +37,13 @@ public partial class ArgonSongProgressBar : SongProgressBar
 
         private double trackTime => (EndTime - StartTime) * Progress;
 
+        private float relativePositionX;
+
+        private InputManager? inputManager;
+
+        public LocalisableString TooltipText => $"{(relativePositionX > 0 ? (EndTime - StartTime) * relativePositionX / DrawWidth : relativePositionX > DrawWidth ? EndTime : 0).ToEditorFormattedString()}"
```

Comment:
This updates with far too much precision. Rounding should be applied to round % to 1% and time to at least seconds. Looks bad right now when scrubbing.

Reply:
Would .1% be neat and a bit accurate at the same time? Seeming so with both short and long maps.